locales/el/editor.properties

```diff
@@ -105,8 +105,12 @@ CMD_TOGGLE_QUICK_DOCS=Γρήγορα έγγραφα
 DND_UNSUPPORTED_FILE_TYPE=μη υποστηριζόμενος τύπος αρχείου
```

Comment:
Before I review this again, you need to fix these changes that aren't related to your patch.  Let me know when you've done all that.